src/ui/EditorMouseWheelMessageFilter.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023, Andrea Anzani <andrea.anzani@gmail.com>
+ * All rights reserved. Distributed under the terms of the MIT license.
+ */
+#ifndef EditorMouseWheelMessageFilter_H
+#define EditorMouseWheelMessageFilter_H
```

Comment:
I started using #pragma once here, so we don't have to change it if we ever rename the file